README.md

```diff
@@ -279,6 +279,17 @@ janusgraph-mount | chown: changing ownership of '/etc/opt/janusgraph/janusgraph.
 ...
 ```
 
+## Docker Tagging Policy
+
+We are ensure following tagging policy for our Docker images:
+
+| Tag            | Support level |
+| -------------- | ------------- |
+| latest         | latest JanusGraph release; no breaking changes guarantees; base layer upgrades, e.g. operating system or java; |
+| 0.x            | newest patch level version of JanusGraph; expect breaking changes; base layer upgrades, e.g. operating system or java; |
```

Comment:
What does it mean to expect breaking changes? If someone asks for JanusGraph 0.5, the only thing that can change is that they will get 0.5.z versions for some z, which is the latest patch-level version (as you said), but that should be backwards-compatible, no?

Other than OS / Java updates, what possible breaking changes would there be?

Reply:
In the normal case this is true, but we already had some breaking changes in patch level versions: https://docs.janusgraph.org/changelog/#tinkerpop-upgrade-from-341-to-344